.claude/skills/github/scripts/issue/get_issue_context.py

```
@@ -72,26 +79,43 @@ def main(argv: list[str] | None = None) -> int:
     try:
         issue_data = json.loads(result.stdout)
     except json.JSONDecodeError:
-        error_and_exit("Failed to parse issue JSON", 3)
+        write_skill_error(
+            "Failed to parse issue JSON",
+            3,
+            error_type="ApiError",
+            output_format=fmt,
+            script_name="get_issue_context.py",
+        )
+        raise SystemExit(3)
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)